pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -207,3 +209,51 @@ func (r *Reconciler) updateStatus(desired *eventingv1alpha1.KnativeEventing) (*e
 	existing.Status = desired.Status
 	return r.KnativeEventingClientSet.OperatorV1alpha1().KnativeEventings(desired.Namespace).UpdateStatus(existing)
 }
+
+// Delete obsolete resources from previous versions
+func (r *Reconciler) deleteObsoleteResources(manifest *mf.Manifest, instance *eventingv1alpha1.KnativeEventing) error {
+	resource := &unstructured.Unstructured{}
+
+	resource.SetNamespace("knative-eventing")
+
+	// 0.13
```

Comment:
I suggest you change the comment to `remove old resources at 0.12`, because 0.13 can make us confused that these resources are in 0.13

Reply:
Done, thx for the suggestion